unix/README.md

```diff
@@ -47,24 +47,29 @@ CPU name
     Or follow the installation instructions on [Git's web
     site](https://git-scm.com/downloads)
 
-2. **`virtualenv`**
+2. **`pipenv`**
 
     ```
-    # Debian/Ubuntu
+    # Debian 10+/Ubuntu 20.04+
     sudo apt-get update
-    sudo apt-get install virtualenv
+    sudo apt-get install pipenv
 
-    # Fedora
+    # Fedora 32+
     sudo dnf -y update
-    sudo dnf -y install python-virtualenv
+    sudo dnf -y install pipenv
 
-    # RHEL/CentOS
-    sudo yum update
-    sudo yum install python-virtualenv
+    # RHEL 7
+    sudo yum install rh-python38
+    scl enable rh-python38 bash
```

Comment:
Awesome addition. Can verify from experience this is required on RHEL. Maybe it was the VM setup I was using, but I noticed it had to be run on each shell session in order for it to work.

Reply:
Yes, you need to enable software collections every time you activate a new shell, or you could *permanently enable* it (in `~/.bashrc`):

```
source scl_source enable rh-python36
```

More details: https://developers.redhat.com/blog/2018/08/13/install-python3-rhel/#scl-permanent

I'm glad they improved the situation in RHEL 8 (i.e `sudo yum install python3`)

Reply:
Alright awesome makes sense. I'd say a quick note of that here will help everyone in the future. To avoid having to rediscover or know that.

Reply:
Added the Troubleshooting section with that info.